custom_components/tuya_local/devices/drexma_et7aw_thermostat.yaml

```diff
@@ -0,0 +1,324 @@
+name: Drexma WiStat Thermostat
```

Comment:
Top level name should be generic, as other brands may also match this config (eg: just "Thermostat")